test/fixtures/mson/issue-689.apib

```diff
@@ -0,0 +1,22 @@
+# API name
+
+# Group Visits
+
+## Visits [/visits]
+
+### Create a Visit [POST]
+
++ Response 200 (application/json)
+    + Attributes (Visit)
+
+# Data Structures
+
+## ContactInfo (object)
+
++ email: june@email.com (string)
```

Comment:
Please use `example.com`, it's a reserved domain name, `june@email.com` could be a legit persons email address which we probably don't want to use in our tests.

[from RFC2606](https://tools.ietf.org/html/rfc2606)

```suggestion
+ email: user@example.com (string)
```